### DISCUSSION TITLE
Hotkey to switch Playback and Mic together?

### DISCUSSION THREAD

**Discussion by mylesuk** [Belphemur/SoundSwitch#964] [Q&A] [answered]
Hi is there a way to have both the Playback device and recording device switch with one hotkey? Would be useful with gaming headsets.

**Comment by Belphemur** [comment, marked as answer]
You can go into the settings, go into Profiles tab and setup a profile for that :)

**Reply by Crabmeat1432** [reply]
I"m confused as to the purpose of the default communications device in this screen. Can you clarify it please?

![image](https://user-images.githubusercontent.com/123743766/217979560-c2366381-db31-4bb6-98cc-7d602d6084f3.png)

**Reply by Belphemur** [reply]
So Windows understand that when you're using a communication app like Zoom, Discord etc you might want a different audio device than the one for the rest of the system. That device will be automatically used by the app when you left it on System Default.

Like you'd have a Bluetooth headset for your Zoom calls and speaker for your music.

This let you set that up, to be able to have a profile where you'd have different device for communication and for media.

If you don't care having a different device for communication, no need to set anything there. SoundSwitch will by default set the main device as communication.

**Comment by Crabmeat1432** [comment]
I understand, thank you.
